homeassistant/components/device_tracker/icloud.py

```diff
@@ -79,8 +80,11 @@ def setup_scanner(hass, config: dict, see, discovery_info=None):
     username = config.get(CONF_USERNAME)
     password = config.get(CONF_PASSWORD)
     account = config.get(CONF_ACCOUNTNAME, slugify(username.partition('@')[0]))
+    max_interval = config.get(CONF_MAX_INTERVAL, 30)
```

Comment:
Is not validated -> https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#3-configuration

Reply:
fixed

Reply:
Default should be set in the `PLATFORM_SCHEMA`.

Reply:
Agree, fixed that. Re-ran all tests and checks. Tested manually that defaults values do take effect.